### PR TITLE
Wiki: Add Advantages And Features Page

### PR DESCRIPTION
**Pull request by RoboticMind**
Creates a page that discusses general benefits of Gridcoin over other cryptocurrencies as well as comparing to other relevant cryptocurrencies individually  

Ideally should be merged alongside #249 since the FAQ links to this page and will now be seen more often

### CODE REVIEW

File: wiki/advantages-and-features.md
Comment (barton2526) [inline wiki/advantages-and-features.md:76]: Clicking the link to this header isn't working for some reason (the others are)

File: wiki/advantages-and-features.md
Comment (barton2526) [inline wiki/advantages-and-features.md:79]: Capitalization of ByteBall

File: wiki/advantages-and-features.md
Comment (barton2526) [inline wiki/advantages-and-features.md:81]: The correct plural here is "Bytes" or "Obyte bytes"

File: wiki/advantages-and-features.md
Comment (barton2526) [inline wiki/advantages-and-features.md:46]: I would prefer "Why is..."

File: wiki/advantages-and-features.md
Comment (barton2526) [inline wiki/advantages-and-features.md:83]: see above

File: wiki/advantages-and-features.md
Comment (barton2526) [inline wiki/advantages-and-features.md:111]: This is fine as long as we make a note to update this whenever the whitelist is changed

File: wiki/advantages-and-features.md
Comment (barton2526) [inline wiki/advantages-and-features.md:129]: this sentence is very awkward

File: wiki/advantages-and-features.md
Comment (barton2526) [inline wiki/advantages-and-features.md:133]: too many uses of "more"

File: wiki/advantages-and-features.md
Comment (barton2526) [inline wiki/advantages-and-features.md:166]: not explained

File: wiki/advantages-and-features.md
Comment (RoboticMind) [reply wiki/advantages-and-features.md:76]: Ah oops I accidentally put comparis***i***on instead of comparison there on that link

File: wiki/advantages-and-features.md
Comment (RoboticMind) [reply wiki/advantages-and-features.md]: Fixed that. I went with "Bytes"

File: wiki/advantages-and-features.md
Comment (RoboticMind) [reply wiki/advantages-and-features.md]: Is this better `Have to overcome the chicken and egg problem for adoption`?

File: wiki/advantages-and-features.md
Comment (RoboticMind) [reply wiki/advantages-and-features.md]: Is this less repetitive? `With Gridcoin more can be rewarded such as fighting disease, finding 
      asteroids, and many other types of science`

File: wiki/advantages-and-features.md
Comment (RoboticMind) [reply wiki/advantages-and-features.md]: Changed it to `Why Gridcoin Is Different`. I think that might be a bit better

File: wiki/advantages-and-features.md
Comment (RoboticMind) [reply wiki/advantages-and-features.md]: Is this any better? `Any expansion of Primecoin to other types of computations would need to fit a Proof of Work algorithm. These algorithms need abilities that many scientific problems lack`

File: wiki/advantages-and-features.md
Comment (RoboticMind) [reply wiki/advantages-and-features.md:111]: Hmm actually I am able to pull directly from the whitelist.yml file so it would only need to be updated in one place with  `{{ site.data.whitelist.projects.size }}`. I'm going to do a PR real quick to update the whitelist and then add that I think

**Comment by RoboticMind** [comment]
Taking this off of draft status, but still waiting for Pythonix's review before merging

File: wiki/advantages-and-features.md
Comment (Pythonix) [inline wiki/advantages-and-features.md:11]:
```suggestion
One of the main advantages of Gridcoin is that crunching ("mining")
```

File: wiki/advantages-and-features.md
Comment (Pythonix) [inline wiki/advantages-and-features.md:15]:
```suggestion
regardless of the number of Gridcoin you earn, crunching alone achieves something 
```

File: wiki/advantages-and-features.md
Comment (Pythonix) [inline wiki/advantages-and-features.md:125]:
```suggestion
* Primecoin can only find prime numbers and no other scientific computations
```

**Review by Pythonix — CHANGES_REQUESTED** [review]
I am not a native speaker so my corrections might be incorrect.

File: wiki/advantages-and-features.md
Comment (RoboticMind) [reply wiki/advantages-and-features.md]: To clarify, both are grammatically correct. However using "of" sounds a bit more natural, so I implemented this

File: wiki/advantages-and-features.md
Comment (RoboticMind) [reply wiki/advantages-and-features.md]: Oops not sure how I missed this one. Fixed that

File: wiki/advantages-and-features.md
Comment (Pythonix) [reply wiki/advantages-and-features.md]: Thanks! Didn't know that.

File: wiki/advantages-and-features.md
Comment (RoboticMind) [reply wiki/advantages-and-features.md:125]: I think I'm going to leave "do" in there to help clarify, but I will add "other" in there. I think both "computation" or "computations" are correct, oddly enough. I don't really know why that is. English grammar is weird.

**Comment by RoboticMind** [comment]
@barton2526 Whenever you get the chance, can you merge this?

**Comment by barton2526** [comment]
After further inspection, the correct capitalization is "Obyte".

**Comment by RoboticMind** [comment]
@barton2526 whenever you get the chance, can you merge this?

**Pull request merged**
